### DISCUSSION TITLE
scenes in bs 1.29.1

### DISCUSSION THREAD

**Discussion by ACMDatabase** [kinsi55/CS_BeatSaber_Camera2#110] [Q&A]
Hi,
I'm using bs 1.29.1 and the latest camera2 for that release, is there anyway to get camera collections(scenes) in this version? beatsaber+ has a chat command that lets you switch scenes and I would love to have that feature. Am I doing something wrong or would this feature be possible in the future? Camera2 is definetly one of the best mods for bs and I cant imagine how streaming would be without it.

**Comment by kinsi55** [comment]
Custom scenes already exist in that version and I know that people do use it with bs+ chat integration so you probably haven't set it up correctly

**Reply by ACMDatabase** [reply]
I really would like to get that working but I need to make them some how. Is there a guide or example?

**Reply by kinsi55** [reply]
https://github.com/kinsi55/CS_BeatSaber_Camera2/wiki/Scenes#custom-scenes

**Reply by ACMDatabase** [reply]
I've searched for hours, thank you :)

**Reply by ACMDatabase** [reply]
Ok I understand how to do it now, your a life saver!
